src/main/resources/changesets/changelog-consent-2023-06-08-add-translated-data-use.xml

```diff
@@ -0,0 +1,17 @@
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+  <changeSet id="2023-06-08-add-translated-data-use" author="connorlbark">
+    <addColumn tableName="dataset">
+      <column name="translated_data_use" type="text"/>
+    </addColumn>
+    <sql>
+      UPDATE dataset
+      SET translated_data_use = (SELECT c.translated_use_restriction AS translated_data_use
+                                 FROM consents c
+                                        INNER JOIN consent_associations ca ON ca.consent_id = c.consent_id
+                                 WHERE ca.dataset_id = dataset_id
+                                 ORDER BY c.create_date DESC LIMIT 1)
+    </sql>
```

Comment:
One concern I have - the value that is stored by `translated_use_restriction` and thus now `translated_data_use` is just free-form text. Why not format it like `/translate/summary` like we do in elastic search? It would make it easier to parse into objects while still have the descriptions available to display to users, and give more options on how to display the data use to users.

Reply:
This is an interesting idea, but not backwards compatible with existing string translations. Let's discuss.

Reply:
We can leave it as a string here and do some work in the reducer to determine if it can be a `DataUseSummary` object. When populating it in future PRs, we can query Ontology for the correct value.

Reply:
I'm not sure I follow - keep it as a string for now and in the future replace it? If so, I'm ok with that, it might be awkward in the intermediary period, though